src/script/standard.cpp

```diff
@@ -221,6 +221,7 @@ bool ExtractDestination(const CScript& scriptPubKey, CTxDestination& addressRet)
     return false;
```

Comment:
same commit:

Should probably remove the comment. "Multisig txns have more than one address..."